index.js

```diff
@@ -103,8 +103,8 @@ module.exports = {
       'components/dialog/dialog-theme.scss'
     ];
 
-    var pathBase = this.project.addonPackages['ember-paper'].path;
-    var angularMaterialPath =  'node_modules/angular-material-source/src';
+    var pathBase = this.project.nodeModulesPath;
```

Comment:
Do you mind explaining the reason for this change?
Does `this.project.nodeModulesPath` equal `this.project.addonPackages['ember-paper'].path + 'node_modules'`?